project-base/app/src/Model/Order/Order.php

```diff
@@ -200,6 +201,18 @@ public function getPaymentTransactions(): array
         return $this->paymentTransactions->getValues();
     }
 
+    /**
+     * @return bool
+     */
+    public function canBePaid(): bool
+    {
+        if ($this->paymentTransactions->count() >= 2) {
+            return false;
+        }
```

Comment:
For me it is not clear why. Can you please elaborate?

Reply:
It's requested to work that way - see the acceptance criteria